collect_app/src/main/AndroidManifest.xml

```diff
@@ -146,16 +146,11 @@
             </intent-filter>
         </activity>
         <activity
-            android:name=".activities.FormManagerList"
-            android:label="@string/app_name" />
-        <activity
             android:name=".activities.FormDownloadList"
             android:label="@string/app_name" />
         <activity
-            android:name=".activities.DataManagerList"
-            android:label="@string/app_name" />
-        <activity
             android:name=".activities.FileManagerTabs"
+            android:configChanges="orientation|screenSize"
```

Comment:
Yes I know what it is but I removed that and everything worked well so what exactly is it for?